### ISSUE TITLE
Remove PQL query endpoint and response format after 0.9.0 release

### ISSUE THREAD

**Issue by xiangfu0** [apache/pinot#7430]
Dear Community,

TL;DR, Pinot will remove PQL query endpoint and response format after 0.9.0 release, only SQL endpoint is supported afterward.

Apache Pinot has been supporting the Calcite SQL query endpoint from 0.3.0 (Mar 2020) and deprecating the PQL query endpoint for more than a year.

Now we are seeing more obstacles to maintain two code paths for PQL & SQL for adding new functionalities, query optimization, behavior handling, etc.

So we decide to remove the PQL query path and endpoint. If you are still on the PQL query path, please migrate to SQL endpoint: https://docs.pinot.apache.org/users/user-guide-query/querying-pinot

Current timeline is driven by the next Apache Pinot release:

1. Apache Pinot 0.9.0 release (ETA Release date: Oct/Nov 2021) is the last official Pinot release that supports PQL.
2. Apache Pinot master branch will remove PQL support once the 0.9.0 release is cut, and no PQL support from the 0.10.0 release.

**Comment by xiangfu0** [comment]
This is huge! Big congrats!

**Comment by xiangfu0** [comment]
Pinot deprecates PQL fully, only SQL endpoint is supported moving forward from 0.11.0 release.
